nautobot/core/testing/filters.py

```diff
@@ -81,19 +84,18 @@ class FilterTestCase(BaseFilterTestCase):
         #       ["filter1"],
         #       ["filter2", "field2__name"],
         #   ]
-        generic_filter_tests = []
+        generic_filter_tests: ClassVar[Iterable]
+
+        def setUp(self):
+            for attr in ["queryset", "filterset", "generic_filter_tests"]:
+                if not hasattr(self, attr):
+                    raise NotImplementedError(f'{self} is missing a value for required attribute "{attr}"')
+            super().setUp()
```

Comment:
Changed these three attributes to abstract, and fail/abort test now if they're not defined by the subclass. I'd like to do the same for our other generic test classes that have required attributes.